packages/common/src/components/Filter/EnumFilter.tsx

```diff
@@ -116,56 +119,58 @@ export const EnumFilter = ({
   filterId,
   showFilter = true,
   resolvedLanguage,
-  hasInlineFilter = false,
-}: FilterTypeProps & InlineFilter) => {
-  const [isExpanded, setExpanded] = useState(false);
+}: FilterTypeProps) => {
+  const [isOpen, setIsOpen] = useState(false);
   const { uniqueEnumLabels, onUniqueFilterUpdate, selectedUniqueEnumLabels } = useUnique({
     supportedEnumValues,
     onSelectedEnumIdsChange,
     selectedEnumIds,
     resolvedLanguage,
   });
 
-  const deleteFilter = (label: string | ToolbarChip | SelectOptionObject): void =>
+  const deleteFilter = (label: string | ToolbarChip): void =>
     onUniqueFilterUpdate(selectedUniqueEnumLabels.filter((filterLabel) => filterLabel !== label));
 
-  const hasFilter = (label: string | SelectOptionObject): boolean =>
+  const hasFilter = (label: string): boolean =>
     !!selectedUniqueEnumLabels.find((filterLabel) => filterLabel === label);
 
-  const addFilter = (label: string | SelectOptionObject): void => {
+  const addFilter = (label: string): void => {
     if (typeof label === 'string') {
       onUniqueFilterUpdate([...selectedUniqueEnumLabels, label]);
     }
   };
 
-  const options = uniqueEnumLabels.map((label) => <SelectOption key={label} value={label} />);
-
-  const onFilter: (
-    event: React.ChangeEvent<HTMLInputElement> | null,
-    textInput: string,
-  ) => React.ReactElement[] | undefined = (_event, textInput) => {
-    if (textInput === '') {
-      return options;
-    }
-    const filtered = options.filter((item) => {
-      return item.key.toString().toLowerCase().includes(textInput.toLowerCase());
-    });
-    return filtered;
+  const onToggleClick = () => {
+    setIsOpen((isOpen) => !isOpen);
   };
 
-  const onSelect: (
-    event: SelectEventType,
-    value: SelectValueType,
-    isPlaceholder?: boolean,
-  ) => void = (_event, value, isPlaceholder) => {
-    if (isPlaceholder) {
-      return;
-    }
-    hasFilter(value) ? deleteFilter(value) : addFilter(value);
+  const onSelect = (
+    _event: React.MouseEvent<Element, MouseEvent> | undefined,
```

Comment:
no need for React , use named import

Reply:
@metalice 
I intentionally left is like that.
It's not working when omitting, there is the interface MouseEvent and there is the React.mouseEvent and when you remove the React, it doesn't recognize. Instead of renaming...I think it's better to leave as is.

I can see that in the `kubevirt-plugin` project it's also the situation and you use React.mouseEvent as is..

Reply:
U can import it from react lib. Named import. Then omit the React.

Reply:
Why you prefer this renaming over using React on that situation? 
Isn't it more clear that way - to which callback you are referring?